src/EditorFeatures/TestUtilities/Workspaces/TestWorkspaceFixture.cs

```diff
@@ -31,12 +31,12 @@ public TestWorkspace GetWorkspace(ExportProvider exportProvider = null)
 
         public TestWorkspace GetWorkspace(string markup, ExportProvider exportProvider = null, string workspaceKind = null)
         {
-            if (TryParseXElement(markup, out var workspaceElement) && workspaceElement.Name == "Workspace")
+            if (markup.TrimStart().StartsWith("<Workspace>"))
```

Comment:
Is it intentional that this is case sensitive?

Reply:
It was not; I guess the XML consumption code assumes case sensitivity of our XML syntax, so probably best to make this case insensitive so a mistake that way doesn't end up running into the same problem I did.